bokehjs/src/lib/core/types.ts

```diff
@@ -29,20 +29,13 @@ export type Size = {
 }
 
 export type Box = {
-  left: number
-  top: number
+  x: number
+  y: number
```

Comment:
whats the impetus for changing the convention here? `top` and `left` are much more descriptive to me, and match css names

Reply:
Because it's not given that `x,y` is actually `top,left`, it depends on the usage.